proto/server.proto

```diff
@@ -140,6 +140,37 @@ message SignUpRequest {
   Meta meta = 5;
 }
 
+message CreateFolderRequest {
+  string path = 1;
+  string name = 2;
+  Meta meta = 3;
+}
+message ListFilesRequest {
+  string path = 1;
+  Meta meta = 2;
+}
+message DeleteFileRequest {
+  string path = 1; // Full path, including the file name and extension
+  Meta meta = 2;
+}
+message DownloadFileRequest {
+  string path = 1; // Full path, including the file name and extension
+  Meta meta = 2;
+}
+message UploadFileRequest {
```

Comment:
Which fields will be present in the subsequent requests?

Reply:
path, name, payload, meta.
Do you want to keep a uuid also as a field?

Reply:
only send the payload

Reply:
Ok